.riot/requirements/1084483.txt

```diff
@@ -10,7 +10,7 @@ certifi==2022.12.7
 charset-normalizer==2.1.1
 click==7.1.2
 coverage[toml]==7.2.2
-flask==1.1.4
+flask==2.3.2
```

Comment:
This is a pretty significant version bump. Looking at the [riotfile.py](https://github.com/DataDog/dd-trace-py/blob/1.x/riotfile.py#L974) this dependency should be pinned to `flask~=1.0`. We should update this bot to respect riot configurations (or disable it from updating packages in .riot/requirements directory).